errbot/backends/slack.py

```diff
@@ -555,11 +555,15 @@ def send_card(self, card: Card):
         if isinstance(card.to, RoomOccupant):
             card.to = card.to.room
         to_humanreadable, to_channel_id = self._prepare_message(card)
-        attachment = {}
+        attachment = {
+            'mrkdwn_in': [],
+        }
         if card.summary:
             attachment['pretext'] = card.summary
+            attachment['mrkdwn_in'].append('pretext')
```

Comment:
Huh, this is telling slack that markdown is in the 'pretext' field; if that's errbot markdown that's fine.

```
By default bot message text will be formatted, but attachments are not. To disable formatting on a non-user message, set the mrkdwn property to false. To enable formatting on attachment fields, set the mrkdwn_in array on each attachment to the list of fields to process.
```

https://api.slack.com/docs/message-formatting